### PR TITLE
Jionghao: Add Experience and About Me Section

### PR DESCRIPTION
**Pull request by Jhwu3**

### CODE REVIEW

**Comment by Jhwu3** [comment]
image
image

File: js/src/personal/jionghao/components/About.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { Title, Text, Image } from '@mantine/core'
+import styles from './About.module.css'
+
+export function About() {
```
Comment (arklian) [inline js/src/personal/jionghao/components/About.tsx:4]: nit: You might want to make each page its own folder under jionghao. Components should likely be things used between pages.

File: js/src/personal/jionghao/components/Experience.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
+import { Title, Text, Tabs, List } from '@mantine/core'
+import styles from './Experience.module.css'
+
+export function Experience() {
```
Comment (arklian) [inline js/src/personal/jionghao/components/Experience.tsx:4]: nit: Might want to break this up a bit more later. Would make sense to make this a folder as well.

**Review by arklian — APPROVED** [review]
Nice!

File: js/src/personal/jionghao/components/About.tsx (hunk 1 above)
Comment (Jhwu3) [reply js/src/personal/jionghao/components/About.tsx:4]: ok got it

**Pull request merged**